apps/design-land/src/app/article/article.component.html

```diff
@@ -6,59 +6,43 @@ <h2>About</h2>
 	<p>The article component can be used on any content pages that displays large blocks of text-driven information. It is meant to be used as a standalone element. It should not be nested inside any other components that may change the background color from the anticipated one. In the event that you must nest the article inside another component, please ensure that you set the article's background color to the default body color.</p>
 
 	<h2>Headings</h2>
-	<h2>This is a h2 heading.</h2>
-	<h3>This is a h3 heading.</h3>
-	<h4>This is a h4 heading.</h4>
-	<h5>This is a h5 heading.</h5>
-	<h6>This is a h6 heading.</h6>
+	<design-land-example-viewer example="article-headings"></design-land-example-viewer>
+
 
 	<h2>Article Lead</h2>
 	<p daffArticleLead>Lead is used for the opening paragraph of an article. It should be used to provide a summary or leading information for an article. Lead is a custom directive of article and is not a native element selector. To use it, add <code>daffArticleLead</code> to a paragraph (<code>&lt;p&gt;</code>)</p>
+	<design-land-example-viewer example="article-lead"></design-land-example-viewer>
+
 
 	<h2>Article Meta</h2>
 	<p daffArticleMeta>Meta is used if there is metadata information about your article (i.e. author name, date, etc). Lead is a custom directive of article and is not a native element selector. To use it, add <code>daffArticleMeta</code> to a paragraph (<code>&lt;p&gt;</code>)</p>
```

Comment:
Description/usage instruction should be using the `<p>` tag without its own directive attached to it.

Reply:
Fixed in b39de07ab474a4355c1c1f0ee530875c0e9a107d